.github/workflows/maven.yml

```diff
@@ -25,3 +24,10 @@ jobs:
       run: mvn -version
     - name: Build with Maven
       run: mvn -B package
+    - name: API testing with Maven
```

Comment:
It's possible that the jar file created on line 26 won't be available for the mvn call on line 28. If that's the case, we'll need to condense these to a single mvn call, i.e.,

`mvn -B package -Prun-api-tests`